oaknational/Oak-Web-Application
# Fix/work with us seo

## Description

- updates worth-with-us page to take SEO props from cms->pageData
- unskips tests in about-us pages
- adds *.fixtures.ts to jest ignore pattern

## Issue(s)

Fixes #949 

## How to test

1. Go to {cloud link}/about-us/work-with-us
2. check seo props ... should not be the defaults, but whatver is set in Sanity


## Checklist
